flocker/common/functional/test_script.py

```diff
@@ -18,6 +18,12 @@
 from eliot import Message
 from eliot.testing import assertContainsFields
 
+try:
+    from eliot.journald import JournaldDestination
+except ImportError as e:
+    # This platform doesn't have journald.
+    JournaldDestination = None
+
```

Comment:
:question: Since `JournaldDestination` is only used to determine whether to skip, perhaps it would be better to save a boolean. I would suggest `journald_available`, but that will be easily confused with `_journald_available()`.

Perhaps the best thing to do is to _also_ rename `_journald_available` so that the distinction between "can query journald" and "can log to journald" is more clear.